docs/PublicAPIsAndMore.md

```diff
@@ -70,41 +67,40 @@ SHiPS contains the following attributes:
 
     By default, SHiPS shows the progress. However an module author can decide to not enable the builtin progress by setting BuiltinProgress to false.
 
-     ``` PowerShell
+    ```powershell
     [SHiPSProvider(BuiltinProgress=$false)]
     ```
 
 ## Public Parameters
 
 The following properties are exposed to SHiPS' derived classes as protected members.
 
-
-`1. Name`: provider's node name
+- `Name`: provider's node name
 
   As above mentioned, you can pass in the node name to base class such as
 
-  ``` PowerShell     
+  ```powershell
   Compute([string]$name) : base ($name)
   {
   }
   ```
   or
 
-  ``` PowerShell
+  ```powershell
   Compute ([string]$name)
   {
      $this.Name = $name;
   }
-  ```   
+  ```
+
+- `ProviderContext`: provider's data
 
-`2. ProviderContext`: provider's data
   - `DynamicParameters`
   As above mentioned, an author can define dynamic parameters for their providers.
   To process whether a user uses these dyanmic parameters, author can access them via the following mechanism.
 
-    ``` PowerShell
+    ```powershell
     $this.ProviderContext.DynamicParameters
-
     ```
 
     For more usages, see [dynamic parameters  example][ds].
```

Comment:
The "dynamic parameters example" link is broken

Reply:
fixed. Will update the PR